### PR TITLE
Hay nos vidrios MTY

### PR DESCRIPTION
**Pull request by barbaragabriela**
Barbara moved to NYC

### CODE REVIEW

File: oontan.geojson [hunk 1]
```diff
@@ -74,14 +74,14 @@
     {
       "type": "Feature",
       "properties": {
-        "city": "Monterrey",
+        "city": "New York",
```
Comment (ixai) [inline oontan.geojson]: Pero ahora hay dos New York, mejor agregate al otro no?

Comment (thewarpaint) [reply oontan.geojson]: +1

https://github.com/thewarpaint/oo/blob/40427e9104f3690fa384caa71911164653a17a30/oontan.geojson#L102-L115

Comment (barbaragabriela) [reply oontan.geojson]: Queria poner mi punto exacto </3

**Comment by Madrigal** [comment]
pls merge

**Comment by thewarpaint** [comment]
pls respect my creative process

**Pull request merged**